web_src/js/index.js

```diff
@@ -755,6 +755,15 @@ async function initRepository() {
     });
   }
 
+  // Repo Creation
+  if ($('.repository.new.repo').length > 0) {
+    $('input[name="gitignores"], input[name="license"]').on('change', () => {
+      const gitignores = $('input[name="gitignores"]').val();
+      const license = $('input[name="license"]').val();
+      $('input[name="auto_init"]')[0].checked = Boolean(gitignores || license);
```

Comment:
```suggestion
      if (gitignores || license) {
          $('input[name="auto_init"]').prop('checked', true);
      }
```

Reply:
Ah, this fixes the code to what was in the original commit. @silverwind -- did you have a reason for changing this behavior? Otherwise, I'm inclined to commit this change.

Reply:
Those `prop('checked')` can be commited, getting a boolean was the intention.

Regarding this condition, I suggest this (so it will also unset, even thought those inputs are currently not clearable):

```js
$('input[name="auto_init"]').prop('checked', gitignores || license);
```

Reply:
I don't think it should unset if it's set, as we don't know users intentions. While if he checks one of these we know he wants to initialize

Reply:
Yeah I guess user could have manually clicked the checkbox before unselecting (it's actually possible), fine with me.